documentation/modules/cruise-control/proc-automating-rebalances.adoc

```diff
@@ -0,0 +1,188 @@
+// Module included in the following assemblies:
+//
+// assembly-cruise-control-concepts.adoc
+
+[id='proc-automating-rebalances-{context}']
+= Triggering auto-rebalances when scaling clusters
+
+[role="_abstract"]
+Set up auto-rebalancing to automatically redistribute topic partitions when scaling a cluster.
+You can scale a Kafka cluster by adjusting the number of brokers using the `spec.replicas` property in the `Kafka` or `KafkaNodePool` custom resource used in deployment. 
+When auto-rebalancing is enabled, the cluster is rebalanced without further intervention.
+
+* After adding brokers, topic partitions are redistributed across the new brokers.
+* Before removing brokers, partitions are moved off the brokers being removed.
+
+Auto-rebalancing helps maintain balanced load distribution across Kafka brokers during scaling operations, depending on how the rebalancing configuration is set up.
+
+Scaling operates in two modes: `add-brokers` and `remove-brokers`.
+Each mode can have its own auto-rebalancing configuration specified in the `Kafka` resource under `spec.cruiseControl.autoRebalance` properties. 
+Use the `template` property to specify a predefined `KafkaRebalance` resource, which serves as a rebalance configuration template.
+If a template is not specified in the `autorebalance` configuration, the default Cruise Control rebalancing configuration is used.
+You can apply the same template configuration for both scaling modes, use different configurations for each, or enable auto-rebalancing for only one mode.
+If `autorebalance` configuration is not set for a mode, auto-rebalancing will not occur for that mode.
+
+The template `KafkaRebalance` resource must include the `strimzi.io/rebalance-template: true` annotation. 
```

Comment:
This is not true anymore btw. You have to add annotation:
```java
strimzi.io/rebalance : template
```
right @ppatierno ?

Reply:
No, it changed the other way around ... from `strimzi.io/rebalance : template` to `strimzi.io/rebalance-template: "true"` ... but the `"true"` should be likely used here as well.

Reply:
> right @ppatierno ?

No as Jakub said, it's the other way around. I already asked Paul to fix it as he did.

Reply:
Thanks. I've kept as true, but added the string quotes